### DISCUSSION TITLE
swiper 9.* does not support SSR

### DISCUSSION THREAD

**Discussion by chenpengxiang1993** [nolimits4web/swiper#6389] [Q&A]
### Check that this is really a bug

- [X] I confirm

### Reproduction link

https://www.mi.com/fr/store

### Bug description

I am upgrading from Swiper 6.8.4 to Swiper 9.0.3, our website need SSR, Swiper 6 colud use SSR, But I found Swiper 8 or Swiper 9 does not support SSR,  now what shoul i do, i want upgrading to Swiper 9, But it doesn't support my needs now

### Actual Behavior

when i use our SSR service publish some page, i get this message, i hope it can work but not

Error [ERR_REQUIRE_ESM]: Must use import to load ES Module: /~/node_modules/swiper/react/swiper-react.js
require() of ES modules is not supported.
require() of /~/node_modules/*/node_modules/swiper/react/swiper-react.js from /~/node_modules/*/lib/src/blocks/Swiper/index.js is an ES module file as it is a .js file whose nearest parent package.json contains "type": "module" which defines all .js files in that package scope as ES modules.
Instead rename swiper-react.js to end in .cjs, change the requiring code to use import(), or remove "type": "module" from /~/node_modules/*/node_modules/swiper/package.json.

    at Module._extensions..js (internal/modules/cjs/loader.js:1080:13)
    at Object.newLoader [as .js] (/~/node_modules/pirates/lib/index.js:104:7)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (/~/node_modules/*/lib/src/blocks/Swiper/index.js:9:15)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Module._compile (/~/node_modules/pirates/lib/index.js:99:24)
    at Module._extensions..js (internal/modules/cjs/loader.js:1092:10) {
  code: 'ERR_REQUIRE_ESM'
}

### Swiper version

9.0.3

### Platform/Target and Browser Versions

macOS、Windows、Cordova、iOS、Android、Chrome

### Validations

- [X] Follow our [Code of Conduct](https://github.com/nolimits4web/swiper/blob/master/CODE_OF_CONDUCT.md)
- [X] Read the [docs](https://swiperjs.com/swiper-api).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.
- [X] Make sure this is a Swiper issue and not a framework-specific issue

### Would you like to open a PR for this bug?

- [x] I'm willing to open a PR

**Comment by saostad** [comment]
I have the same problem with Nextjs and Sveltekit projects.
the app loads with all the slides on it, then the slider loads and hides the slides.
Here is a link to the Sveltekit test repo:  
https://github.com/saostad/swiper-sveltekit
DEMO https://swiper-sveltekit.vercel.app/

**Comment by HomyeeKing** [comment]
![image](https://github.com/nolimits4web/swiper/assets/49113249/58a01028-e610-49e6-9ebe-39ec030640a3)

smoothScroll not support ssr
